react/react1/week3/todo-app-week3/src/components/Form.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
```

Comment:
Here: input and deadline state would be better to be managed by this component as it is this one that needs and uses it.